tests/execute/escaping/main.fmf

```diff
@@ -0,0 +1,13 @@
+framework: shell
+duration: 1s
+require: []
+
+/escaping/01-basic quote's error:
```

Comment:
'/escaping' is already part of the test name so no need to duplicate it (directory name is part of test name)

```suggestion
/01-basic quote's error:
```

Reply:
Yeah, I overlooked the core tests. It makes sense to move it there. Please see the current state.